### DISCUSSION TITLE
Need discount for Shopify apps

### DISCUSSION THREAD

**Discussion by zeeshanhaider6236dev** [Kyon147/laravel-shopify#144] [Ideas]
https://www.shopify.com/partners/blog/app-billing-improvements

New features are:

App Subscription Discounts
Prorated App Credit on Cancellation
Trial Extension and Proration Options on App Subscription Charges

Can anyone add into this package ?

**Comment by alimud1** [comment]
Yes its must have in this package

**Comment by kurakin-oleksandr** [comment]
Maybe it will be helpful: https://github.com/Kyon147/laravel-shopify/discussions/111

**Reply by alimud1** [reply]
is it possible like this way on app purchase inside shopify 

https://cdn.shopify.com/s/files/1/0533/2089/files/app-billing-improvements-approve-charges.jpg?v=1650297299

**Comment by Kyon147** [comment]
@zeeshanhaider6236dev @alimud1 feel free to open a PR if you wish to bring this functionality into the package.
